### ISSUE TITLE
numeric accel keys blocked by VimFx

### ISSUE THREAD

**Issue by mjauvin** [akhodakivskiy/VimFx#703]
Steps to reproduce:

1. go to http://marc.jauvin.com/access.html
2. type "1", "2", or "3" on the keyboard
3. An alert window should tell you which link would be opened

Whenever VimFx is enabled, Javascript keyboard events for 0-9 are not received by the Javascript keyboard event handler

VimFx version: 0.12.0 
Firefox version: 44.0.2
OS: Linux
Keyboard language: English US

**Comment by lydell** [comment]
Duplicate of #646.

See also: https://github.com/akhodakivskiy/VimFx/blob/master/documentation/options.md#counts_enabled
